main/src/ca/uwaterloo/flix/language/phase/jvm/BackendType.scala

```diff
@@ -70,7 +64,7 @@ sealed trait BackendType extends VoidableType {
   /**
     * A string representing the erased type. This is used for parametrized class names.
     */
-  val toErasedString: String = this match {
+  def toErasedString: String = this match {
```

Comment:
what's the reason for this change?

Reply:
I thought I had changed it to `val` so I wanted to revert it, but I see that's not the case.